pkgs/applications/audio/bucklespring/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+, legacy ? false
+, libinput
+, libudev
```

Comment:
libudev is undefined

Reply:
odd, builds here with and without it (i think i added it before libinput)
but pkgs.libudev is defined... (-> udev -> systemd)